### DISCUSSION TITLE
Vocabulary Builder notation

### DISCUSSION THREAD

**Discussion by thunkN** [koreader/koreader#11645] [Ideas]
Is there a way to have a word that is added to Vocabulary Builder be marked somehow? I know you can also 'highlight' the word, but I reserve that function for quotes and passages. Thank you!

**Comment by jonnyl2** [comment]
Not specifically, but why not just use a different highlight style, like underline or invert?

**Reply by thunkN** [reply]
I'd like for that kind of marking to be applied to just the single word added to the Vocabulary Builder. The problem, at least for me, is that 'highlighting' the single word adds to the Bookmarks page where highlighted quotes are stored. I'd prefer to keep vocab words and quotes to be separated. I know this is just a personal preference so I'm not expecting it to be troubleshooted. Just an Idea. 😅

**Reply by jonnyl2** [reply]
In case you didn't know: you can filter by highlight style in the Bookmarks screen. So you could select one or the other and they wouldn't get mixed. (Press hamburger menu in top left corner.)

**Reply by thunkN** [reply]
That does help a lot when I'm on the device, thank you! But when I export the highlights to a computer it then lists the highlighted vocabulary words along with the annotations. Again, I know this is just a personal preference so I'm not expecting an overhaul of the feature.
